pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>token-macro</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>1.3.0</version>
```

Comment:
Ahhh so docker-plugin shades docker-java... that makes things somewhat easier, in that you are not going to worry about docker-plugin conflicting